### PR TITLE
Added a /reload command

### PR DESCRIPTION
**Pull request by ghost**
just wanted to add a few commands to help!

### CODE REVIEW

**Comment by Wild1145** [comment]
The standard reload command as part of Spigot will be more than enough. Please focus on open issues and not random commands.

File: src/com/superiornetworks/icarus/commands/Command_reload.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandParameters(name = "reload", description = "Reload the Server.", usage = "/reload <command>", rank = ICM_Rank.Rank.TELNET)
+public class Command_reload
+{
+  {
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_reload.java:14]: This has no corresponding '}'.

File: src/com/superiornetworks/icarus/commands/Command_reload.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        ICM_Util.bcastMsg("[IcarusMod] is reloading. Please wait!", ChatColor.YELLOW);
+
+        server.reload();
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_reload.java:19]: There is no variable defined with the name 'server'.

**Comment by Hockeyfan360** [comment]
I'm not trying to be overly-critical, but it is fairly obvious that this was not written in any IDE or tested. All pull requests should be tested by yourself before you submit

**Pull request closed without merging**